music21/stream/base.py

```diff
@@ -6830,6 +6831,27 @@ def makeAccidentals(
                 if e.tie is not None and e.tie.type != 'stop':
                     tiePitchSet.add(e.pitch.nameWithOctave)
 
+                # handle this note's ornaments' otherPitches
+                for orn in e.expressions:
+                    if not isinstance(orn, expressions.Ornament):
+                        continue
+
+                    orn.resolveOtherPitches(e.pitch)
+                    if not orn.otherPitches:
+                        continue
+
+                    orn.updateAccidentalDisplay(
+                        pitchPast=pitchPast,
+                        pitchPastMeasure=pitchPastMeasure,
+                        alteredPitches=alteredPitches,
+                        cautionaryPitchClass=cautionaryPitchClass,
+                        cautionaryAll=cautionaryAll,
+                        overrideStatus=overrideStatus,
+                        cautionaryNotImmediateRepeat=cautionaryNotImmediateRepeat,
+                        lastNoteWasTied=False)
+
+                    pitchPast += orn.otherPitches
```

Comment:
even from this I still have no idea what .otherPitches is.  Can't otherPitches be None?

Reply:
I started out with `orn.pitches`, which seems reasonable (what pitches do you play when you play this ornament), but what I really want is "what pitches _besides the main note's pitch_ do you play when you play this ornament", so I came up with `orn.otherPitches`, which makes sense in context, but doesn't provide any context itself.  How about `orn.ornamentalPitches`?  They are the pitches that are added by the ornament.

Reply:
Ah, how about `ornamentalPitches`?  only a few more letters, but it makes clear that it doesn't include the main pitch

Reply:
Done.  It's a really good name.